packages/framework/azure-service-utils/src/generateToken.ts

```diff
@@ -28,7 +28,11 @@ export function generateToken(
     }
 
     // Current time in seconds
-    const now = Math.round((new Date()).getTime() / 1000);
+    const now = Math.round(Date.now() / 1000);
+
+    if (documentId === undefined) {
+        throw new Error("documentId cannot be undefined");
+    }
```

Comment:
I don't understand why this function has a nullable documentId parameter... Could I make it required? Or use a different type for the claims?